blog/models.py

```diff
@@ -0,0 +1,26 @@
+"""Models for the blog app."""
+
+from django.conf import settings
+from django.db import models
+from django.utils import timezone
+
+
+class Post(models.Model):
+    """A class to represent a post."""
+
+    author = models.ForeignKey(
+        settings.AUTH_USER_MODEL, on_delete=models.CASCADE,
+    )
+    title = models.CharField(max_length=200)
+    text = models.TextField()
+    created_date = models.DateTimeField(default=timezone.now)
+    published_date = models.DateTimeField(blank=True, null=True)
+
+    def publish(self):
+        """Save the post entry."""
```

Comment:
I'd say something like "mark the post published" since this is what this does on the high level.

Reply:
Marking as resolved.